DBCD/DBCDBuilder.cs

```diff
@@ -83,7 +83,17 @@ internal DBCDBuilder(Locale locale = Locale.None)
                 var columnDefinition = databaseDefinition.columnDefinitions[fieldDefinition.name];
                 bool isLocalisedString = columnDefinition.type == "locstring" && locStringSize > 1;
 
-                var fieldType = FieldDefinitionToType(fieldDefinition, columnDefinition, localiseStrings);
+
+                Type fieldType;
+                if (fieldDefinition.isRelation && fieldDefinition.isRelation)
```

Comment:
Probably this?

```suggestion
                if (fieldDefinition.isRelation && fieldDefinition.isNonInline)
```

Reply:
That'a actually supposed to be `isArray`, but yes, an oof.

Reply:
Actually, scratch that, not needed at all. Oof has been oofed.